client/src/component/Jobs/index.js

```diff
@@ -9,7 +9,7 @@ const Jobs = () => {
 
   return (
     <div>
-      {user.user_types_id === 2 ? (
+      {user.user_types_id === 2 || user.user_types_id === 1 ? (
```

Comment:
Good idea to allow admin users to see this information as well. we could probably do something similar on jobs.